tests/testthat/test-4IO.R

```diff
@@ -40,6 +40,19 @@ test_that("Importing dada2 objects yield TreeSummarizedExperiment objects", {
     expect_s4_class(me, "TreeSummarizedExperiment")
 })
 
+test_that("Importing Mothur files yield SummarizedExperiment objects", {
+    
+    counts <- system.file("extdata", "mothur_example.shared", package = "mia")
+    taxa <- system.file("extdata", "mothur_example.cons.taxonomy", package = "mia")
+    taxa2 <- system.file("extdata", "mothur_example.taxonomy", package = "mia")
+    meta <- system.file("extdata", "mothur_example.design", package = "mia")
+    se <- loadFromMothur(counts, taxa, meta)
+    se2 <- loadFromMothur(counts, taxa2, meta)
+    
+    expect_s4_class(se, "SummarizedExperiment")
+    expect_s4_class(se2, "SummarizedExperiment")
```

Comment:
Can you add a few tests on the dimension? This ist just to make sure, that the `SE` actually conforms to the expectations (An `SE` can in theory have 0 rows or 0 columns).

Please check the following
- dimensions
- assays present
- rowData present + correct dimension
- colData present + correct dimension
Thanks

Reply:
Added tests